frontend/src/metabase/api/task.ts

```diff
@@ -0,0 +1,34 @@
+import type {
+  ListTasksRequest,
+  ListTasksResponse,
+  Task,
+} from "metabase-types/api";
+
+import { Api } from "./api";
+
+export const taskApi = Api.injectEndpoints({
+  endpoints: builder => ({
```

Comment:
`providesTags` are missing in `listsTasks` and `getTask`.
We can skip it for `getTasksInfo`.

Reply:
Or maybe we can skip them for all endpoints... I mean, it will work the same way anyway since we don't have manipulating endpoints. I think we're good :+1:

Reply:
I intentionally didn't add anything to this list since there isn't any mechanism on the FE that would invalidate the cache.

Reply:
https://github.com/metabase/metabase/pull/41452/commits/cd7024cb9291aba34378b2f5dcbd1ec7952c1636